docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -0,0 +1,70 @@
+# Signalen and the Standard for Public Code
+
+## [Code in the open](https://standard.publiccode.net/criteria/code-in-the-open.html)
+
+* Current backlog and bug-tracking is not public. Some issues referenced in code and commits are not easy to locate, like "SIG-1525". Transition to public bug-tracker?
+* Can the classification system be completely open sourced? How can we address risks and possibilities regarding (of theoretical) leaking of information. Maybe add anonymized MOR datasets or redo the classification algorithm with public available MOR data sets. Can a basic Common Ground version be made?
+* Citizens are not authenticated. Data is not cleaned up for sharing (privacy purposes). https://github.com/Amsterdam/signals/blob/master/docs/api-design.md#publiek-en-ongeauthenticeerd
+
+## [Bundle Policy And Source Code](https://standard.publiccode.net/criteria/bundle-policy-and-code.html)
+
+* Making the state machine configurable would enable Signalen to adapt to policy changes within Amsterdam and also enable replicators to customise Signalen for their workflows.
+
+## [Create reusable and portable code](https://standard.publiccode.net/criteria/reusable-and-portable-codebases.html)
+
+* There are several dependencies on Amsterdam services. Replicators need this to be configurable.
+* Technical roadmap is not published.
+* [Workflow](https://github.com/Amsterdam/signals/blob/master/api/app/signals/apps/signals/workflow.py) is currently hard-coded. Names of key states can be found elsewhere, need abstraction plan.
+* Email messages are [hard-coded](https://github.com/Amsterdam/signals/blob/master/api/app/signals/apps/email_integrations/core/messages.py).
+* Separate UI text from API model classes: [example](signals/api/app/signals/apps/signals/models/history.py).
+* Extended category questions currently live in front-end code, extract and make plug-able.
+* Will "sia" prefixes be meaningful to replicators? Or will aliasing or renaming something less AMS specific be better?
```

Comment:
We will revisit this at a later time, for now we will keep the name SIA.

Reply:
Removed from the code